storage/remote/queue_manager.go

```diff
@@ -1821,6 +1753,7 @@ type offLenPair struct {
 
 type rwSymbolTable struct {
 	symbols         []byte
+	strings         []string
 	symbolsMap      map[string]offLenPair
 	symbolsMapBytes map[string]uint32
 }
```

Comment:
we can remove some of what's here in `rwSymbolTable` now that we're not going with the slice of bytes approach right?